examples/source_examples/oauth2_with_token_refresh/readme.md

```diff
@@ -0,0 +1,40 @@
+# OAuth2 Refresh Token Hubspot Connector Example
+
+> NOTE: We have Fivetran connector for HubSpot that users can integrate directly on the dashboard [here](https://fivetran.com/docs/connectors/applications/hubspot#hubspot). You can use this example as a reference for the cases where you want to connect to a custom OAuth2 source that requires access token refresh.
+
+## Prerequisites
+
+1. If you don't have a HubSpot account, follow HubSpot's [instructions](https://developers.hubspot.com/docs/guides/apps/public-apps/overview) to create one.
+
+2. Create a developer account and a HubSpot app with scopes and redirect URL. See HubSpot's [OAuth documentations](https://developers.hubspot.com/docs/reference/api/app-management/oauth) for details.
+
+3. Use the ClientId, ClientSecret and RedirectUrl from the developer account, and paster it in the Postman app inside Authorization->OAuth2.0 as shown in the below SS:
```

Comment:
```suggestion
3. Paste the **ClientID**, **Client Secret** and **Redirect URL** values from the developer account in the (<list_the_corresponding_fields_here>  on the **Authorisation** tab in Postman:
```
I don't see Callback URL mentioned while it's on the screenshot, and the the values in dev account and fields in Postman should be mapped.